accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -909,18 +951,25 @@ non-breaking.
     to all other libraries, regardless of the versions of those libraries.
     "Ignorance of the law is no defense."*
 
-*   We would like to add modifiers to some classes in platform (i.e. `dart:`)
+*   We would like to add modifiers to some classes in platform (i.e., `dart:`)
     libraries when this feature ships. But we would also like to not immediately
     break existing code. To avoid forcing users to immediately migrate,
     declarations in pre-feature libraries can ignore *some*
     `base`, `interface` and `final` modifiers on *some* declarations
-    in platform libraries.
+    in platform libraries, and can mix in non-`mixin` classes from platform libraries,
```

Comment:
Discussion of interactions between language versions are split between the section titled "Mixin restrictions" and here.  Can they be unified?  It took me a long time to realize that the things addressed in the other section were in fact addressed, and then an even longer period of flipping back and forth between the two areas to convince myself that all of the combinations were covered.  Combining them in one place would make it much clearer.

Reply:
I think I have all the discussion of *why* platform libraries are treated specially here, and the actual rules integrating that behavior directly in their specification, not as an appended exception.

Hope it works.